src/api/composition-api-dependency-injection.md

```diff
@@ -1,106 +1,108 @@
-# Composition API: <br>Dependency Injection {#composition-api-dependency-injection}
+# Composition API: <br>Inyección de Dependencias {#composition-api-dependency-injection}
 
 ## provide() {#provide}
 
-Provides a value that can be injected by descendent components.
+Proporciona un valor que puede ser inyectado por componentes descendientes.
 
-- **Type**
+- **Tipo**
 
   ```ts
   function provide<T>(key: InjectionKey<T> | string, value: T): void
   ```
 
-- **Details**
+- **Detalles**
 
-  `provide()` takes two arguments: the key, which can be a string or a symbol, and the value to be injected.
+  `provide()` toma dos argumentos: la key, que puede ser una cadena de texto o un symbol, y el valor a inyectar.
 
-  When using TypeScript, the key can be a symbol casted as `InjectionKey` - a Vue provided utility type that extends `Symbol`, which can be used to sync the value type between `provide()` and `inject()`.
+  Cuando se utiliza TypeScript, la clave puede ser un symbol convertido como `InjectionKey` - un tipo de utilidad proporcionado por Vue que extiende `Symbol`, que se puede utilizar para sincronizar el tipo de valor entre `provide()` e `inject()`.
 
-  Similar to lifecycle hook registration APIs, `provide()` must be called synchronously during a component's `setup()` phase.
+  Al igual que las APIs de registro de los hooks de ciclo de vida, `provide()` debe ser llamado de forma sincrónica durante la fase `setup()` de un componente.
 
-- **Example**
+- **Ejemplo**
 
   ```vue
   <script setup>
   import { ref, provide } from 'vue'
   import { fooSymbol } from './injectionSymbols'
 
-  // provide static value
+  // proporcionar un valor estático
   provide('foo', 'bar')
 
-  // provide reactive value
+  // proporcionar un valor reactivo
   const count = ref(0)
   provide('count', count)
 
-  // provide with Symbol keys
+  // proporcionar con llaves Symbol
   provide(fooSymbol, count)
   </script>
   ```
 
-- **See also**:
-  - [Guide - Provide / Inject](/guide/components/provide-inject)
-  - [Guide - Typing Provide / Inject](/guide/typescript/composition-api#typing-provide-inject)
+- **Véase también**:
+  - [Guía - Provide / Inject](/guide/components/provide-inject)
+  - [Guía - Escritura de Provide / Inject](/guide/typescript/composition-api#typing-provide-inject)
 
 ## inject() {#inject}
 
-Injects a value provided by an ancestor component or the application (via `app.provide()`).
+Inyecta un valor proporcionado por un componente antepasado o por la aplicación (a través de `app.provide()`).
```

Comment:
Antepasado suena raro, quizás ¿un componente de nivel superior?

Reply:
nivel superior it is